lib/minio/header_signer.rb

```diff
@@ -16,22 +16,22 @@
 class Minio::HeaderSigner
   SERVICE_NAME = "s3"
   def build_headers(method, uri, body, creds, region)
+    date = Time.now.utc
```

Comment:
since we're refactoring, let's name this `timestamp`, it makes more sense.

Reply:
The Python version used 'date'. If you're not that strong, I can keep them the same. This simplifies porting other functions.

https://github.com/minio/minio-py/blob/fbcfca1b96c8919a496f8420dba0be139091364b/minio/api.py#L241